content/en/docs/collector/management.md

```diff
@@ -0,0 +1,97 @@
+
+This document describes how you can manage your OpenTelemetry collector
+deployment at scale.
+
+## Basics
+
+Telemetry collection at scale requires a structured approach to manage agents.
+Typical agent management tasks include:
+
+1. Querying the agent information and configuration. The agent information can
+   include its version, operating system related information, or capabilities.
+   The configuration of the agent refers to its telemetry collection setup, for
+   example, the OpenTelemetry collector
+   [configuration][otel-collector-configuration].
+1. Upgrading/downgrading agents and management of agent-specific packages,
+   including the base agent functionality and plugins.
+1. Applying new configurations to agents. This might be required because of
+   changes in the environment or due to policy changes.
+1. Health and performance monitoring of the agents, typically CPU and memory
+   usage and also agent-specific metrics, for example, the rate of processing or
+   backpressure-related information.
+1. Connection management between a control plane and the agent such as handling
+   of TLS certificates (revocation and rotation).
+
+Not every use case requires support for all of the above agent management tasks.
+In the context of OpenTelemetry task _4. Health and performance monitoring_ is
+ideally done using OpenTelemetry.
+
+## OpAMP
+
+Observability vendors and cloud providers offer proprietary solutions for agent
+management. In the open source observability space, there is an emerging
+standard that you can use for agent management: Open Agent Management Protocol
+(OpAMP).
+
+The [OpAMP specification][opamp-spec] (status: Beta) defines how to manage a
+fleet of telemetry data agents. These agents can be [OpenTelemetry
+collectors][otel-collector], Fluent Bit or other agents in any arbitrary
+combination.
+
+> **Note** The term "agent" is used here as a catch-all term for OpenTelemetry
+> components that respond to OpAMP, this could be the collector but also SDK
+> components.
+
+OpAMP is a client/server protocol that supports communication over HTTP and over
+WebSockets:
+
+- The **OpAMP server** is part of the control plane and acts as the
+  orchestrator, managing a fleet of telemetry agents.
+- The **OpAMP client** is part of the data plane. The client side of OpAMP can
+  be implemented in-process, for example, as the case in [OpAMP support in the
+  OpenTelemetry collector][opamp-in-otel-collector]. The client side of OpAMP
+  could alternatively be implemented out-of-process. For this latter option, you
+  can use a supervisor that takes care of the OpAMP specific communication with
+  the OpAMP server and at the same time controls the telemetry agent, for
+  example to apply a configuration or to upgrade it. Note that the
+  supervisor/telemetry communication is not part of OpAMP.
+
+Let's have a look at a concrete deployment:
+
+![OpAMP example setup](/img/docs/OTel_OpAMP.svg)
```

Comment:
I know that you've moved the image already, but I'd like to ask that you relocate the image closer to where it is being used, and rename the file to follow the website's file naming conventions. So move and rename:

- `/img/docs/OTel_OpAMP.svg`

to 

- `/content/en/docs/collector/img/opamp.svg`

Then update the reference in the file.
```suggestion
![OpAMP example setup](../img/opamp.svg)
```